Assets/Python/Victory.py

```diff
@@ -950,9 +950,8 @@ def checkTurn(iGameTurn, iPlayer):
 				lose(iRussia, 0)
 				
 		if isPossible(iRussia, 0):
-			if teamRussia.isHasTech(iRailroad):
-				if isConnectedByRailroad(iRussia, Areas.getCapital(iRussia), lSiberianCoast):
-					win(iRussia, 0)
+			if isConnectedByRailroad(iRussia, Areas.getCapital(iRussia), lSiberianCoast):

```

Comment:
I moved the railroad tech check to the isConnectedByRailroad() function. The calculation will still not be done if the player does not have the railroad tech. But now these calculations will also be skipped when opening the victoryscreen while the player does not have the railroad tech.